apps/dsaparam.c

```diff
@@ -128,10 +129,15 @@ int dsaparam_main(int argc, char **argv)
         }
     }
 
-    /* Optional arg is bitsize. */
+    /* Optional args is bitsize and q bitsize. */
```

Comment:
Optional args **are** bitsize and q bitsize

Reply:
fixed